app/src/main/java/fr/neamar/kiss/dataprovider/AppProvider.java

```diff
@@ -125,31 +128,119 @@ public void requestResults(String query, Searcher searcher) {
             return;
         }
 
-        FuzzyScore fuzzyScore = FuzzyFactory.createFuzzyScore(this, queryNormalized.codePoints);
+        boolean flexibleFuzzy = prefs.getBoolean("enable-fuzzy-search", false);
```

Comment:
Can you please encapsulate new search same way as other search algorithm?

This way it's just working for apps, but there are lots of other places using fuzzy search like shortcuts, contacts (surname, last name, nickname, ...) and some others. Also refresh of results needs it at some point.

See `FuzzyFactory`, there you can return new implementation of `FuzzyScore` including levenshtein.